content/howto/geodesic/authenticate-with-leapp.md

```diff
@@ -0,0 +1,182 @@
+- `sa-east-1` if in South America,
+- `eu-north-1` if in Europe,
+- `af-south-1` if in Africa,
+- `ap-south-1` if in Asia, or
+- `ap-southeast-2` if in Australia or Antarctica.
+
+#### Credentials
+
+You will have to have some way of authenticating to AWS already set up, and you
+need to know the nature of your credentials. Whoever gave you access to AWS
+should be able to tell you what kind of access you have. It should be one of the following:
+
+- **Federated Role** (recommended). This means you log into something you use to
+identify yourself to a lot of places (usually refered to as Single Sign-On
+or SSO), and it is set up to log you into AWS with a specific IAM Role as well. 
+At a company, this is usually Google Workspaces (formerly GSuite),
+Okta, or Active Directory. This is also known as a SAML IdP.
+- **AWS SSO**. This means your company has configured AWS as a Single Sign-On
+provider. This is AWS _as_ a Single Sign-On provider, allowing you to access
+multiple _permission sets_ within AWS, not using some
+other Single Sign-On provider to sign in to AWS as a single IAM Role. 
+Sadly this is made even
+more confusing by the fact that even though your company has set up 
+AWS _as_ a Single Sign-On provider, you still may be using your company's
+primary SSO provider to authenticate to AWS SSO. 
+- **AWS IAM User**. This is the older way of authenticating to AWS, with a basic
+username (actually an email address) and password to log into the AWS console,
+and a long-lived "Access Key" for API access. If you are going to use this
+method, we strongly recommend you enable multi-factor authentication (MFA).
+
+Based on which kind of credentials you have, you will need to gather different
+information with which to configure Leapp. Whoever is in charge of setting up
+your access to AWS should be able to give you the information you need.
+
+##### Federated Role
+
+If using Federated Login, you will need
+- Your IdP Single Sign-On URL. 
+  - For Google Workspaces, it looks something like
+    `https://accounts.google.com/o/saml2/initsso?idpid=C0asdfasdfal&spid=12344321`.
+  - For Okta, it looks something like `https://company.okta.com/app/company_samlidp_1/Hka1abcke6h4P1WQr5d7/sso/saml`. 
+  - Most importantly,it should not be confused with the AWS Single Sign-On URLs
+  like these (**do not use these**)
+    - `https://signin.aws.amazon.com/saml` (this is used by your IdP admin)
+    - `https://my-sso-portal.awsapps.com/start` (this means you are using AWS SSO)
+- The AWS Identity Provider ARN assigned by AWS to your IdP. Something like
+  - `arn:aws:iam::123434211234:saml-provider/company-name`
+- Your assigned/authorized AWS IAM Role ARN. Something like
+  - `arn:aws:iam::123434211234:role/role-name`
+
+##### AWS SSO
+
+If using AWS SSO, you will need:
+
+- Your AWS SSO "start URL", also known as your "portal URL". It should be 
+very close to:
+  - `https://something.awsapps.com/start`
+
```

Comment:
Also, the region in which AWS Single Sign-On has been deployed is needed in order to work.

The company implementing AWS Single Sign-On has to choose a specific aws region to deploy the service.

It's a needed field in Leapp

Reply:
Thank you. I will add that.